src/widget.cpp

```diff
@@ -1849,9 +1865,16 @@ void NWidgetSpacer::FillNestedArray(NWidgetBase **, uint)
 {
 }
 
-void NWidgetSpacer::Draw(const Window *)
+void NWidgetSpacer::Draw(const Window *w)
 {
-	/* Spacer widget is never visible. */
+	/* Spacer widget is never normally visible. */
+
+	if (_draw_widget_outlines && this->current_x != 0 && this->current_y != 0) {
+		/* Spacers indicate*/
```

Comment:
```suggestion
		/* Spacers indicate */
```

Reply:
Oops, I was writing a comment and didn't finish it off...